src/LaravelRequestDocsToOpenApi.php

```diff
@@ -111,19 +115,23 @@ protected function attributeIsFile(string $rule): bool
         return str_contains($rule, 'file') || str_contains($rule, 'image');
     }
 
-    protected function makeQueryParameterItem(string $attribute, $rule): array
+    protected function makeQueryParameterItem(string $attribute, $rule, array $description): array
     {
         if (is_array($rule)) {
             $rule = implode('|', $rule);
         }
         $parameter = [
-            'name' => $attribute,
-            'description' => $rule,
             'in' => 'query',
+            'name' => $attribute,
+            'description' => $description['description'] ?? $rule,
+            'example' => $description['example'] ?? null,
```

Comment:
Is there any way for us to validate the OpenAI specification?

Reply:
@kitloong is trying to check if the new changes that come to openAPI generated json have been tested using Postman.
I ll test it out, you can skip this comment.